src/Polly.Pcl/project.json

```diff
@@ -1,12 +1,7 @@
 ﻿{
-    "dependencies": {
-        "NuSpec.ReferenceGenerator": {
-            "version": "1.4.0",
-            "type": "build"
-        }
-    },
-    "frameworks": {
-        ".NETPortable,Version=v4.5,Profile=Profile259": { }
-    },
-    "supports": {}
-}
+  "dependencies": {},
+  "frameworks": {
+    ".NETPortable,Version=v4.5,Profile=Profile259": {}
```

Comment:
This profile is supported by .NET Standard 1.0, so it's not necessary any more if you don't want to maintain it separately (see https://docs.microsoft.com/en-us/dotnet/articles/standard/library)